tools/sim/bridge.py

```diff
@@ -211,18 +211,25 @@ def bridge(q):
   gps = world.spawn_actor(gps_bp, transform, attach_to=vehicle)
   gps.listen(lambda gps: gps_callback(gps, vehicle_state))
 
-  def destroy():
-    print("clean exit")
-    imu.destroy()
-    camera.destroy()
-    vehicle.destroy()
-    print("done")
-  atexit.register(destroy)
-
   # launch fake car threads
-  threading.Thread(target=panda_state_function).start()
-  threading.Thread(target=fake_driver_monitoring).start()
-  threading.Thread(target=can_function_runner, args=(vehicle_state,)).start()
+  StoppableThread = namedtuple("StoppableThread", ["thread", "exit_event"])
```

Comment:
no need for this. just use a single event and set it

Reply:
I agree that will work. I chose distinct events to preclude the possibility of a future bug of one thread influencing the flow control of another since the shared event is mutable. If you don't think we need to worry about that much, please confirm, and I'll make the change.

Reply:
yeah, that's really not a concern

Reply:
Thanks for confirming. I'll make the change when I'm back in front of my dev machine.

Reply:
Done.